### PR TITLE
Add final paper

### PR DESCRIPTION
**Pull request by boyinggong**

### CODE REVIEW

**Comment by BenjaminHsieh** [comment]
Need to edit the line limit to 80 characters

File: paper/report_final.tex [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+manually determine a good threshold for a mask we use to isolate more active 
+voxels in the brain. This helps us find beta coefficients for more relevant 
+voxels. We also model and remove the linear and quadratic drift that may be 
+present in the runs. 
+We use subject 2 run 2 to show an example of our outliers results.
+
+\begin{figure}[H]
+    \centering
+        \includegraphics[scale=0.55]{figures/processing.png}
+    \caption{DVARS, Framewise Displacement and Data mean for subject 2 run 2}
```
Comment (BenjaminHsieh) [inline paper/report_final.tex:139]: Please add to the caption on what are the green lines in dvars, from #107

Comment (boyinggong) [inline paper/report_final.tex:139]: @changsiyao do you know what the green line is?

File: paper/report_final.tex [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+Therefore, we compute the neural loss aversion at every voxel by subtracting
+the slope of the gain response from the (negative) slope of the loss response.
+With the neural loss aversion values calculated, we can explore how loss
+aversion affects brain activation when potential gains and losses are
+considered.
+
+\subsubsection{Model Diagnosis for linear regression}
+
+From the results of the QQ plot, we can see that in this randomly picture. The residuals are approximately normal distribution. The green line is the QQ plot of the normal distribution. The blue line is the QQ plot of residuals. They look quite similar. The second plot is the scatter plot of the residuals. We can see that the residuals are approximately equally distributed by 0. This means that the residuals are not correlated to the fitted values. To conclude, the residuals are approximately normal distributed.
+
```
Comment (BenjaminHsieh) [inline paper/report_final.tex:485]: Mention the the residual vs fitted show constant variance

**Comment by changsiyao** [comment]
Green line in dvars and fd are threshold for outliers, 0.5 for both graph. I explained it in the outlier script.
Sorry, I'm not at a computer right now, please fix it for me. Thanks!

File: paper/report_final.tex (hunk 1 above)
Comment (BenjaminHsieh) [inline paper/report_final.tex:139]: isn't it just a least squared fitted curve of the data points?
see comments below

**Comment by BenjaminHsieh** [comment]
I know the green dotted line is the threshold for outliers, but what about the mean solid line in meanSignal plot? Is it a least squares fitted curve?

**Comment by changsiyao** [comment]
The line in mean data is a fitted smooth curve of the bold signals.

File: paper/report_final.tex [hunk 3]
```diff
@@ -0,0 +1,578 @@
...
+We use K-means clustering to detect the homogeneity of regions. We concatenate the betas obtained from linear models and the coordinate of voxels as the clustering vector (dimension = 16 + 3 = 19). Then do cluster analysis for beta gains and beta loss separately to detect regions activated by increasing beta losses and beta gains.
+
+
+\section{Results}
+
+\subsection{Behavioral analysis}
+
+We performed statistical analysis using both Python and R (The original paper use R package to fit the Logistic models). We use the library \emph{scikit-learn} in Python and the \emph{glm} function in \emph{stats} in R to fit the models. Models from two library yields the same results. We shows the plot of the behavioral loss aversion $\lambda$ for every subject (median=1.94, mean=2.18, min=0.99, max=0.75). This result is consistent with that of the paper, which indicate that participants are indifferent to gambles whose gain are approximately twice as the loss. 
+
```
Comment (BenjaminHsieh) [inline paper/report_final.tex:372]: Umm we dont use R to do any analysis right? Ie there is no R code involved. Should probabilty rephase this as using the `statsmodels` and `scipy` module of **Python**, delete the comment about us using R

**Comment by boyinggong** [comment]
Can we merge this now and then do the revision? cuz the travis really takes a long time for this pull request.

**Comment by briankleinqiu** [comment]
Forget about travis. It might be broken. Matthew posted that on piazza

**Comment by boyinggong** [comment]
OK. Sorry I didn't saw that on piazza.

File: paper/report_final.tex (hunk 2 above)
Comment (boyinggong) [inline paper/report_final.tex:485]: Done!

File: paper/report_final.tex (hunk 3 above)
Comment (boyinggong) [inline paper/report_final.tex:372]: I think it doesn't matter to mention this. We do use R to do the exploratory analysis.

Comment (BenjaminHsieh) [inline paper/report_final.tex:372]: lol if we did we need to include the R scipts as supplementary material

File: paper/report_final.tex [hunk 4]
```diff
@@ -0,0 +1,578 @@
...
+    \includegraphics[width=0.53\textwidth]{figures/Regression1/roc_curve.png} 
+    \caption{ROC Curve for all subjects}
+  \end{center}
+\end{wrapfigure}
+
+Following are the model diagnosis:
+
+\begin{itemize}
+\item \emph{Accuracy on the training dataet} We uses the fitted models on the original dataets and compared the estimated class and the true class using the Logistic classifier. The accuracy of Logistic models (for 16 participants, 16 models in total) on the training set yielded a median of 89.78\% (min=80.97\%, max=99.21\%).
+\item \emph{Cross-validation} We did the model evaluation using 10-fold cross-validation for every subject, they are still performing accuracies of a median of 89.86\% (min=79.92\%, max=98.45\%).
```
Comment (BenjaminHsieh) [inline paper/report_final.tex]: Can we specify what accruacy means? see ross' comments #107

**Comment by BenjaminHsieh** [comment]
On your next commit message, include `[ci skip]` anywhere in the message, and travis will be skipped

File: paper/report_final.tex (hunk 4 above)
Comment (boyinggong) [inline paper/report_final.tex]: accuracy is explained in the model part

File: paper/report_final.tex (hunk 1 above)
Comment (BenjaminHsieh) [inline paper/report_final.tex:139]: ok

**Comment by BenjaminHsieh** [comment]
@boyinggong is it ok to merge?

**Comment by boyinggong** [comment]
Yes. I've fixed problems you mentioned.

**Comment by boyinggong** [comment]
Should we delete the section folder?

**Pull request merged**